R/layer.r

```diff
@@ -68,12 +68,13 @@ layer <- function(geom = NULL, stat = NULL,
                   position = NULL, params = list(),
                   inherit.aes = TRUE, check.aes = TRUE, check.param = TRUE,
                   show.legend = NA, key_glyph = NULL, layer_class = Layer) {
+  call_env = caller_env()
```

Comment:
It's just temporary for improving the errors in the layer constructor. When we eventually store it in the layer object it will just be the call stripped of the environment